gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=org.web3j
-version=4.6.0
+version=4.6.4
```

Comment:
Shouldn't this stay at `4.6.5-SNAPSHOT` or such ?

Reply:
Yes, my bad.